jwst/datamodels/schemas/gain.schema.yaml

```diff
@@ -2,6 +2,7 @@ allOf:
 - $ref: referencefile.schema.yaml
 - $ref: subarray.schema.yaml
 - $ref: keyword_gainfact.schema.yaml
+- $ref: bunit.schema.yaml
```

Comment:
I can't remember whether the order of the reference `$ref` has effect on the order of keywords in the FITS header.
And if so if anyone cares if bunit is at the end of a header.

Reply:
BUNIT appears in the SCI extension header, while all the others appear in the primary. Hence ordering is not a concern.